setup.py

```diff
@@ -10,7 +10,7 @@
     long_description = fh.read()
 
 setuptools.setup(
-    name="olxcleaner",
+    name="openedx-olxcleaner",
```

Comment:
Changing the distribution name because `olxcleaner` name is already taken on PyPI

Reply:
I'm happy to take down that package - better for the actively maintained version to be there.

Reply:
Yeah sure, if that's the case I would go ahead and revert the package name in this PR. Thanks the rapid response.

Reply:
Actually, can I just add the open edx account as a collaborator? What would the username be?

Reply:
I found the "edx" user and sent an invite to be a maintainer on the package on pypi. The invite is pending acceptance.

Reply:
@jmbowman If you could help us out here...

Reply:
Sorry for the late reply.  The password I have on file for the edx PyPI user is outdated, you'll probably need an SRE ticket for this.

Reply:
@aht007 @jmbowman Let me know if I can be helpful here...

Reply:
@jolyonb maybe the invitation expired, can you send it again to the `edx` user on PyPI?

Reply:
@feanil Resent.

Reply:
@jolyonb I still don't see it.  Can you try one more time?  or try to add the `openedx` user instead?

Reply:
@feanil Just reinvited edx and openedx

Reply:
Got it! I've accepted the invite as `openedx`.  I'll now remove you as the owner.  Thank you for all the work so far!